homeassistant/helpers/trigger_template_entity.py

```diff
@@ -200,6 +196,15 @@ def restore_attributes(self, last_state: State) -> None:
                 extra_state_attributes[attr] = last_state.attributes[attr]
             self._rendered[CONF_ATTRIBUTES] = extra_state_attributes
 
+    def _render_template_variables(
```

Comment:
The method name is misleading IMHO considering no template rendering happens here.
Maybe just `_get_template_variables` or `_template_variables`?